packages/cli/src/LoadNodesAndCredentials.ts

```diff
@@ -307,6 +315,55 @@ export class LoadNodesAndCredentialsClass implements INodesAndCredentials {
 		}
 	}
 
+	/**
+	 * Whether any of the node's credential types may be used to
+	 * make a request from a node other than itself.
+	 */
+	private supportsProxyAuth(description: INodeTypeDescription) {
+		if (!description.credentials) return false;
+
+		return description.credentials.some(({ name }) => {
+			const credType = this.types.credentials.find((t) => t.name === name);
+			if (!credType) return false;
```

Comment:
maybe we should throw an error here? or at least a big red warning.
otherwise someone with a custom node with an invalid credential name is likely to complain that their node isn't working as expected.

Reply:
I think a `Logger.warn` should be enough, since this won't be breaking the node completely, just disabling the custom api options for that node.
so something like `Failed to load Custom API options for the node "${nodeType}": Unknown credential name "${credentialType}"`

Reply:
Makes sense 👍